src/app/docs/timepicker/timepicker-docs.component.html

```diff
@@ -0,0 +1,20 @@
+<sky-docs-demo-page
+  moduleName="SkyTimepickerModule"
+  moduleSourceCodePath="src/app/public/modules/timepicker/"
+  pageTitle="Timepicker"
+>
+  <sky-docs-demo-page-summary>
+    The timepicker module creates a SKY UX-themed input field and picker for users to enter or select times.
+  </sky-docs-demo-page-summary>
+
+  <!-- <sky-docs-code-examples
+    [packageDependencies]="{
+      '@skyux/datetime': '*'
+    }"
+  >
+    <sky-docs-code-example
+      sourceCodePath="src/app/public/plugin-resources/code-examples/time-picker/basic/"
+    >
+    </sky-docs-code-example>
+  </sky-docs-code-examples> -->
```

Comment:
I created https://github.com/blackbaud/skyux-datetime/issues/173 for porting over the existing guidelines.